climada/engine/unsequa/input_var.py

```diff
@@ -68,40 +68,39 @@ class InputVar():
     --------
 
     Categorical variable function: LitPop exposures with m,n exponents in [0,5]
-
-    >>> import scipy as sp
-    >>> def litpop_cat(m, n):
-    ...     exp = Litpop.from_countries('CHE', exponent=[m, n])
-    ...     return exp
-    >>> distr_dict = {
-    ...     'm': sp.stats.randint(low=0, high=5),
-    ...     'n': sp.stats.randint(low=0, high=5)
-    ...     }
-    >>> iv_cat = InputVar(func=litpop_cat, distr_dict=distr_dict)
+        import scipy as sp
```

Comment:
This file did not need any modification. Why was the docstring style changed?

Reply:
no idea. I reverted the change by replacing the file with the current version of input_var.py on develop.